docs/Working Agreement.md

```diff
@@ -0,0 +1,166 @@
+
+## Work life balance
+
+- Our office hours, when we can expect to collaborate via Microsoft Teams, phone or face-to-face are Monday to Friday
+  10AM - 3PM (PST) when possible.
+- Sometimes emails or IM messages happen when thoughts come to mind and may come outside of working hours. We are not
+  expected to answer emails or IMs past 5PM, on weekends, or when we are on holidays or vacation.
+- We work in different time zones and respect this, especially when setting up recurring meetings.
+- We record meetings when possible, so that team members who could not attend live can listen later.
+
+## Quality and not quantity
+
+- We agree on a Definition of Done for our user stories and sprints and live by it.
+- We follow engineering best practices like the [Code With Engineering
+  Playbook](https://github.com/microsoft/code-with-engineering-playbook)
+- We try to do demos every sprint when appropriate, we don't force a demo to fit just because it is the end of the
+  sprint.
+- Appropriate time to prepare demos is given and an expectation of a demo on a moments notice will be avoided.
+
+## Scrum rhythm (This will be completed after ADS)
+
+We will follow guidance from the Code With Engineering Playbook on [Agile core
+expectations](https://github.com/microsoft/code-with-engineering-playbook/blob/main/docs/agile-development/core-expectations/README.md),
+which provides guidance on the purpose and structure of the ceremonies noted below.
+
+| Activity                       | When                  | Duration    | Who          | Accountable   | Goal                                                                                                                                |
+|--------------------------------|-----------------------|-------------|--------------|---------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| Project Stand-up & Parking Lot | Mon-Fri 10:05 Pacific | 40 min      | Everyone     | Process Lead  | Stand-up: What has been accomplished, next steps, blockers (15 minutes) <br> Parking Lot: Optional extended discussion (25 minutes) |
+| Sprint Review                  | Tuesdays 9:30AM       | 120 minutes | Everyone     | Tech Lead     | Present work done and sign off on user story completion.                                                                            |
+| Sprint Retrospective           | Tuesdays 10:15AM      | 90 minutes  | Everyone     | Process Lead  | Dev Teams shares learnings and what can be improved.                                                                                |
+| Sprint Planning                | Wednesdays 9:30AM     | 90 minutes  | Everyone     | Product Owner | Size and plan user stories for the sprint.                                                                                          |
```

Comment:
I would think Sprint Planning should be the same day as review?

Reply:
Yes, Sprint Planning will be changed to Tuesday and duration will be 60 minutes.